internal/pkg/service/stream/definition/branch.go

```diff
@@ -6,6 +6,7 @@ import (
 
 type Branch struct {
 	key.BranchKey
+	Switchable
 	SoftDeletable
```

Comment:
All Sources in the branch can be disabled at all. 

It is not currently used in the API, but it was in the past designed to support multiple branches.
Currently, only the `main` branch is allow, because full suport of DEV storage is missing in Keboola.

In future: If you create a new DEV branch, all streams should be copied and disabled in the branch until you enable them. In DEV branch, the target is not a production table, but a table in DEV storage.